root/report/RecordingsWithFutureDates.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import Layout from '../layout';
+import formatUserDate from '../utility/formatUserDate';
+
+import RecordingRelationshipList from './components/RecordingRelationshipList';
+import type {ReportDataT, ReportRecordingRelationshipT} from './types';
+
+const RecordingsWithFutureDates = ({
+  $c,
+  generated,
+  items,
+  pager,
+}: ReportDataT< ReportRecordingRelationshipT>): React.Element<typeof Layout> => (
+  <Layout
+    $c={$c}
+    fullWidth
+    title={l('Recordings with dates in the future')}
+  >
+    <h1>{l('Recordings with dates in the future')}</h1>
+
+    <ul>
+      <li>
+        {exp.l(
+          `This report shows recordings with relationships using dates in the
+           future. Those are probably typos (e.g. 2109 instead of 2019).`
+        )}
+      </li>
+      <li>
+        {texp.l('Total releases found: {count}',
```

Comment:
Recordings, or probably better relationships, not releases :)

Reply:
fixed